doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-subprocesses-ref.adoc

```diff
@@ -51,6 +51,13 @@ Reusable sub-processes appear collapsed within the parent process. To configure
 .Reusable sub-process properties
 image::BPMN2/subproc-props.png[A screenshot of sub-process properties]
 
+[NOTE]
+====
+You can open the sub-process in a new editor in {CENTRAL} from the main business process by clicking the *Place order* task and clicking the *Open Sub-process* task icon.
```

Comment:
You can open the sub-process in a new editor in {CENTRAL}  by clicking the *Place order* task in the main process and  then clicking the *Open Sub-process* task icon.

I really don't think you should have the graphic in the note. The graphic is useful. Why not just take all of that out of the note?

Reply:
Done. Thank you :)